verified_model/Site/South/epsg_code.json

```diff
@@ -0,0 +1,74 @@
+{
+    "value": 32719,
+    "type": "int",
+    "file": false,
+    "metadata": {
+        "CTA": {
+            "REFERENCE": {
+                "VERSION": "1.0.0"
+            },
+            "CONTACT": {
+                "ORGANIZATION": "CTAO",
+                "NAME": "CTAO Simulation Team",
+                "EMAIL": null
+            },
+            "PRODUCT": {
+                "DESCRIPTION": "EPSG code. Fixed value for given longitude/latitude.",
+                "CREATION_TIME": "2024-01-11T08:16:11",
+                "ID": null,
+                "DATA": {
+                    "CATEGORY": "CAL",
```

Comment:
Does "CAL" stand for "Calibration"? I am surprised both by the use of a short version and by the fact it is under calibration and not simulation.

Reply:
This is one of the points where the top-level data model documents points towards and non-existing document ("see CTA data identifier document"). For event data, this is `A`, `B`, `C`. Agree that this shouldn't be CAL, but I also don't know if it should be "SIM". Should we indicate it somehow that we don't know? "UNDEFINED"?

Reply:
I tried going over the documents Dima collected [here](https://gitlab.cta-observatory.org/cta-computing/common/config-data-structures/-/wikis/home) and couldn't find anything relevant except the `A`, `B`, `C` you mentioned. So I agree we should set it as `UNDEFINED` for now and fix it later.

Reply:
I am setting it `null`, that should be for now (otherwise I have to change the schema, allowed values are `CAL`, `SIM`, null .